IFC4x3/Properties/s/SignalMessage_0tVDoRl1v74RpBVl3MpeD6/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SignalMessage_0tVDoRl1v74RpBVl3MpeD6" Name="SignalMessage" UniqueId="377cdc9b-bc1e-4711-bccb-7ef0d6ce8346" PrimaryDataType="IfcText" />
```

Comment:
Wondering what the exchange scenario for this is. Do you have a storyline where this was exchanged?

Reply:
No storyline yet. Domain wants to have messages that can be expressed by the signal.